hugo/layouts/partials/site/head.html

```diff
@@ -1,5 +1,11 @@
 <meta charset="utf-8">
 <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+<meta name="description" content="
+{{ with .Description | $.RenderString | plainify }}{{ . }}
+{{ else }}{{ with .Summary }}{{ . | plainify | htmlUnescape | chomp | truncate 160 -}}
+{{ else }}{{ with .Site.Params.description }}{{ . | $.RenderString | plainify }}
+{{ end }}{{ end }}{{ end }}
```

Comment:
is it correct we have 3 end tags here? I though and if else only needed 1? (not sure though..)

Reply:
yes, its needed. https://github.com/gohugoio/hugo/issues/9818 (example)